### PR TITLE
document freeing fixed vars with example

### PR DESCRIPTION
**Pull request by mjburton11**
Ready for review.  

Made example in docs showing and explaining how to free fixed variables.

### CODE REVIEW

**Comment by whoburg** [comment]
test this please

**Comment by bqpd** [comment]
@whoburg, I heard you say you had some more changes on this pr?

File: docs/source/advancedcommands.rst [hunk 1]
```diff
@@ -133,6 +133,24 @@ You can also substitute by just calling the solution, i.e. ``solution(p)``. This
 
 .. _Sweeps:
 
+Freeing Fixed Variables
```
Comment (bqpd) [inline docs/source/advancedcommands.rst]: Should this go a couple of lines up?

File: docs/source/advancedcommands.rst [hunk 2]
```diff
@@ -133,6 +133,24 @@ You can also substitute by just calling the solution, i.e. ``solution(p)``. This
...
+    from gpkit import Variable, Model
+    x = Variable("x")
+    y = Variable("y", 3)
+    m = Model(x, [x >= 1+y, y >= 1])
+    m.solve()
+
+    del m.substitutions["y"]
+    _ = m.solve()
+
+Please note that the ``y.value``, will still be 3.  In other words, after deleted ``y`` in the model, it will solve as if ``y`` is a free variable. If the variable, ``y``, is used in a seperate model, it will still carry the value of 3.
```
Comment (bqpd) [inline docs/source/advancedcommands.rst]: This is well said but a little long; could use some tightening.

File: docs/source/advancedcommands.rst (hunk 1 above)
Comment (whoburg) [inline docs/source/advancedcommands.rst]: done locally, will push soon

**Comment by whoburg** [comment]
test this please

**Comment by whoburg** [comment]
added a commit, ready for review.

**Comment by mjburton11** [comment]
Looks good to me.  The inline comments add a lot.  I'll try to do that for future documentation.

**Comment by whoburg** [comment]
Thanks for adding this, @mjburton11!

**Comment by bqpd** [comment]
:clap:

**Pull request merged**